nyaa/backend.py

```diff
@@ -62,16 +77,14 @@ def _recursive_dict_iterator(source):
 
 
 def _validate_torrent_filenames(torrent):
-    ''' Checks path parts of a torrent's filetree against blacklisted characters,
-        returning False on rejection '''
-    # TODO Move to config.py
-    character_blacklist = [
-        '\u202E',  # RIGHT-TO-LEFT OVERRIDE
-    ]
+    ''' Checks path parts of a torrent's filetree against blacklisted characters
+        and filenames, returning False on rejection '''
     file_tree = json.loads(torrent.filelist.filelist_blob.decode('utf-8'))
 
     for path_part, value in _recursive_dict_iterator(file_tree):
-        if any(True for c in character_blacklist if c in path_part):
+        if path_part.rsplit('.', 1)[0].lower() in FILENAME_BLACKLIST:
+            return False
+        if any(True for c in CHARACTER_BLACKLIST if c in path_part):
```

Comment:
Time to try this new suggestion feature.

```suggestion
        if any(c in path_path for c in CHARACTER_BLACKLIST):
```

Reply:
It's `path_part`, not `path_path`. This code also has nothing to do with the actual proposed change in this PR, so I don't know what you're doing here.

Reply:
I was just making a suggestion with the mentioned new feature from Github. Fixed the typo.